src/i18n/de.json

```
@@ -672,7 +672,9 @@
             "plural": "Durchschnittliche Bewertung: {{rating}} (aus {{count}} Bewertungen)"
         },
         "rss-link": "Abonniere die Kommentare zu diesem Artikel mit Deinem RSS-/Atom-Feed-Reader.",
-        "warning": "Beachte bitte, dass wir ein <em>unabhängiger Datenschutzverein</em> sind und nicht zu dem hier aufgeführten Unternehmen gehören.<br>Solltest Du also Support benötigen oder eine Anfrage stellen wollen, wende Dich bitte direkt an das Unternehmen. Wir können Dir hierbei <strong>nicht</strong> helfen. Danke für Dein Verständnis."
+        "warning": "Beachte bitte, dass wir ein <em>unabhängiger Datenschutzverein</em> sind und nicht zu dem hier aufgeführten Unternehmen gehören.<br>Solltest Du also Support benötigen oder eine Anfrage stellen wollen, wende Dich bitte direkt an das Unternehmen. Wir können Dir hierbei <strong>nicht</strong> helfen. Danke für Dein Verständnis.",
+        "confirm-private-data": "Huch, scheinbar enthält dein Kommentar sensible Daten. Willst du ihn wirklich ÖFFENTLICH posten? Er wird das Unternehmen nie erreichen.",
+        "regex-sus-words": "(bestell)|(retoure)|(paket)|(p(ä|ae)ckchen)"
```

Why the matching groups?

```suggestion
        "regex-sus-words": "bestell|retoure|paket|p(ä|ae)ckchen"
```

---

I might be wrong, but iirc I encountered engines that require groups here.

---

Engines as in browser engines or regex engines (in other languages)? Works fine without the groups in Firefox and Chrome at least.